generator/README.md

```diff
@@ -1,31 +1,51 @@
-# generator
+# sidekick: a tool to generate and maintain Google Cloud SDKs
 
-A tool for generating client libraries.
+`sidekick` automates (or will soon automate) most activities around generating
+and maintaining SDKs for Google Cloud.
 
-## Example Run
+## Example Run with Protobuf
 
-Run the following command from the generator directory:
+You wneed to have `protoc` installed in your path. You can find useful links
+below.
 
-```bash
-go run ./devtools/cmd/generate -language=rust
-```
-
-Alternatively, you can run the protoc command directly:
+This will generate the client library for [Secret Manager] in the
+`generator/testdata/rust/openapi/golden` directory. In future releases most
+options should be already configured in a `.sidekick.toml` file.
 
 ```bash
-go install ./cmd/protoc-gen-gclient
-
-protoc -I testdata/googleapis \
-    --gclient_out=. \
-    --gclient_opt=capture-input=true,language=rust \
-    testdata/googleapis/google/cloud/secretmanager/v1/resources.proto \
-    testdata/googleapis/google/cloud/secretmanager/v1/service.proto
+cd generator
+go run ./cmd -project-root=.. generate \
```

Comment:
nit: consider using the fully qualified install path so that this command can be run from anywhere. Also, command may change from suggestion based on another comment.

```suggestion
go run github.com/googleapis/google-cloud-rust/generator/cmd@latest -project-root=.. generate \
```

Reply:
Done.